python-project-template/.pre-commit-config.yaml.jinja

```diff
@@ -101,17 +104,18 @@ repos:
 {% elif preferred_linter == 'black' %}
     # Analyze the code style and report code that doesn't adhere.
   - repo: https://github.com/psf/black
-    rev: 23.1.0
+    rev: 23.7.0
     hooks:
-      - id: black
+      - id: black-jupyter
```

Comment:
Why `black-jupyter` here?

Reply:
This change allows _black_ to format Jupyter notebooks.